docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -2,488 +2,716 @@
 
-  [21:03:55.955 INFO    processor_handlers] registered transaction processor: identity=b'6d2d80275ae280ea', family=sawtooth_settings, version=1.0, namespaces=<google.protobuf.pyext._message.RepeatedScalarContainer object at 0x7e1ff042f6c0>
-  [21:03:55.956 DEBUG   interconnect] ServerThread sending TP_REGISTER_RESPONSE to b'6d2d80275ae280ea'
+   The ``settings-tp`` transaction processor continues to run and to display log
+   messages in its terminal window.
 
-Verifying that the REST API is Running
---------------------------------------
+#. Start the IntegerKey transaction processor, ``intkey-tp-python``.
 
-In order to configure a running validator, the REST API must be running.
-Open a new terminal window and run the following command:
+   #. Open a new terminal (the intkey terminal window). The prompt
+      ``user@intkey$`` shows the commands that should be run in this window.
 
-.. code-block:: console
+   #. Run the following command:
 
-  $ ps aux | grep sawtooth-rest-api
-  sawtooth  2829  0.0  0.3  55756  3980 pts/0    S+   19:36   0:00 sudo -u sawtooth sawtooth-rest-api -v
-  sawtooth  2830  0.0  3.6 221164 37520 pts/0    Sl+  19:36   0:00 /usr/bin/python3 /usr/bin/sawtooth-rest-api -v
-  ubuntu    3004  0.0  0.0  12944   928 pts/4    S+   19:54   0:00 grep -E --color=auto sawtooth-rest-api
+      .. code-block:: console
 
-If necessary, run the following command to start the REST API.
+         user@intkey$ sudo -u sawtooth intkey-tp-python -v
+         [23:07:57 INFO    core] register attempt: OK
 
-.. code-block:: console
+   #. Check the validator terminal window to confirm that the transaction
+      processor has registered with the validator.  A successful registration
+      event produces the following output:
 
-  $ sudo -u sawtooth sawtooth-rest-api -v
+      .. code-block:: console
 
-Configuring the List of Transaction Families (Ubuntu version)
--------------------------------------------------------------
+         [2018-03-14 15:56:35.255 INFO     processor_handlers] registered transaction processor: connection_id=94d1aedfc2ba0575a0e4b4f06be7ff7875703f18817027b463b3772ce2b963adb9902f7ed0bafa50201e6845015f65bac814302bdafbcda6e6698fe1733b9411, family=intkey, version=1.0, namespaces=['1cf126']
 
-One of the on-chain settings is the list of supported transaction families.
-To configure this setting, use the following steps to
-create and submit the batch to change the transaction family settings.
+   The ``intkey-tp-python`` transaction processor continues to run and to
+   display log messages in its terminal window.
 
-In this example, the ``sawset`` command specifies a JSON array
-which creates and submits a batch of transactions containing the
-settings change. This JSON array tells the validator or validator network
-to accept transactions of the following types:
+#. Start the XO transaction processor, ``xo-tp-python``. This transaction
```

Comment:
I would make this an optional step, since it is only needed for the later tutorial.